hooks/image/60-kolibri-content

```diff
@@ -143,4 +153,35 @@ fi
 
 if [ -n "${EIB_ENDLESSKEY_INSTALL_CHANNELS}" ]; then
   prepare_kolibri_env "endless-key"
+  apps_url=$(
+    curl -sSL https://api.github.com/repos/endlessm/kolibri-explore-plugin/releases/latest |
+    jq -r '.assets[] | select(.name == "apps-bundle.zip") | .browser_download_url'
+  )
+  collections_url=$(
+    curl -sSL https://api.github.com/repos/endlessm/endless-key-collections/releases/latest |
+    jq -r '.tarball_url'
+  )
+
+  PLUGIN_LOCATION=`pip3 show kolibri-explore-plugin | grep Location  | awk '{ print $2 }'`
+  APPS_LOCATION="${PLUGIN_LOCATION}/kolibri_explore_plugin"
+  COLLECTIONS_LOCATION="${PLUGIN_LOCATION}/kolibri_explore_plugin/static/collections/"
```

Comment:
I don't think we need to do this. These bundles are only needed at runtime and aren't required if you're just running `importchannel`/`importcontent`.

Reply:
But don't we need to put them in place, so they can be found at runtime? Or do they already come with the Flatpak?

Reply:
The flatpak has them. Otherwise, how would it work if you just install the flatpak?

https://github.com/flathub/org.endlessos.Key/blob/master/modules/endless-key-collections.json
https://github.com/flathub/org.endlessos.Key/blob/master/modules/python3-kolibri-explore-plugin.json

Reply:
That makes sense.. I was confused [by your earlier comment](https://github.com/endlessm/eos-image-builder/issues/117#issuecomment-1688784247), but I guess things were not as well defined at that point.